en/documentation/templates/render.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: home
+title: Render
+sidebar: templates
+lang: en
+subnav: templates_render
+---
+
+# Render
+
+The render function takes a mandatory argument ```action```, that is the method of the action. It supports ```a:b:c``` notation. 
+Example:
+
+```smarty
+{render action="Module:Foo:bar"}
+```
+
+Will executes ```Module\Controller\FooController::barAction``` and print the result.
+
+__You can specify the request method__:
+
+```smarty
+{render action="Module:Foo:bar" method="PUT"}
+```
+
+__You can specify the query parameters__:
+
+```smarty
+{render action="Module:Foo:bar" query="foo=bar&baz=thelia"}

```

Comment:
Do we have to urlencode the parameters ?

Something like `foo="bar"` `baz="thelia"` instead of  `query="foo=bar&baz=thelia"` would be more consistent with other Thelia smarty functions